### DISCUSSION TITLE
Maestro vs detox

### DISCUSSION THREAD

**Discussion by aymane-missouri-nw** [infinitered/ignite#2400] [Q&A]
Why did you migrate to maestro ?

**Comment by oottoohh** [comment]
it still using detox i think

**Reply by aymane-missouri-nw** [reply]
There is a new PR that will take maestro in favor of detox and approved

**Comment by frankcalise** [comment]
We're discussing internally whether or not to make the switch. The reason being is we love the ease of writing e2e tests with Maestro. The PR was created to explore what it would look like
